mail_debrand/__manifest__.py

```diff
@@ -14,6 +14,5 @@
     "license": "AGPL-3",
     "installable": True,
     "depends": ["mail"],
-    "development_status": "Stable",
```

Comment:
Please change it to `Production/Stable` instead

Reply:
@pedrobaeza You mean `Production/Stable` would be a key?

Reply:
It would be the proper value for the key `development_status`.

Reply:
Ok, I will modify the value.